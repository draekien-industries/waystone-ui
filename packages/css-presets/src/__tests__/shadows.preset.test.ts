import { boxShadowCss } from '../shadows.presets';

describe('Box Shadow preset', () => {
  it('should match snapshot', () => {
    expect(boxShadowCss).toMatchSnapshot();
  });
});
