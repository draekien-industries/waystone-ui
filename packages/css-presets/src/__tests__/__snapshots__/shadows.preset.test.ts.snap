// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box Shadow preset should match snapshot 1`] = `
{
  "active": {
    "boxShadow": "medium",
  },
  "base": {
    "boxShadow": "small",
  },
  "focused": {
    "boxShadow": "medium",
  },
  "none": {
    "boxShadow": "none",
  },
}
`;
